README.md

```diff
@@ -74,56 +75,88 @@ The following runtime flags are available:
+This is a history of past and current rules by which positive samples are further filtered and identified as
+'filtered positive'. Note that any rule change requires the `update_filtered_positives` migration be run, as outlined
+in the below relevant section.
 
-The implementation of the current version can be found in [FilteredPositiveIdentifier](./crawler/filtered_positive_identifier.py), with the implementation of previous versions (if any) in the git history.
+The implementation of the current version can be found in [FilteredPositiveIdentifier](./crawler/filtered_positive_identifier.py),
+with the implementation of previous versions (if any) in the git history.
 
 #### Version 1 `v1` - **Current Version**
 
 A sample is filtered positive if:
 
-- it has a positive RESULT
-- it is not a control (ROOT_SAMPLE_ID does not start with 'CBIQA_')
-- all of CH1_CQ, CH2_CQ and CH3_CQ are `None`, or one of these is less than or equal to 30
+* it has a positive RESULT
+* it is not a control (ROOT_SAMPLE_ID does not start with 'CBIQA_')
+* all of CH1_CQ, CH2_CQ and CH3_CQ are `None`, or one of these is less than or equal to 30
 
-More information on this version can be found on [this Confluence page](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/UAT+6th+October+2020).
+More information on this version can be found on [this](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/UAT+6th+October+2020)
+Confluence page.
 
 #### Propagating Filtered Positive version changes to MongoDB, MLWH and DART
 
-On changing the positive filtering version/definition, all unpicked samples stored in MongoDB, MLWH and DART need updating to determine whether they are still filtered positive under the new rules, and can therefore be picked in DART.
-In order to keep the databases in sync, the update process for all is performed in a single manual migration (update_filtered_positives) which identifies unpicked wells, re-determines their filtered positive value, and updates the databases.
+On changing the positive filtering version/definition, all unpicked samples stored in MongoDB, MLWH and DART need
+updating to determine whether they are still filtered positive under the new rules, and can therefore be picked in DART.
+In order to keep the databases in sync, the update process for all is performed in a single manual migration
+(update_filtered_positives) which identifies unpicked wells, re-determines their filtered positive value, and updates
+the databases.
 
 Usage (inside pipenv shell):
 
     python run_migration.py update_filtered_positives
 
-The process does not duplicate any data, instead updating existing entries.
+The process does not duplicate any data, instead updates existing entries.
+
+### Migrating legacy data to DART
+
+When the Beckman robots come online, we need to populate the DART database with the filtered positive samples that are
+available physically. This can be achieved using the 'update_dart' migration.
```

Comment:
This can also be used similarly to the existing MLWH migration: if a DART insert process fails, you will see a critical exception for the file in the Lighthouse-UI. After addressing reason for failure, run between relevant timestamps to re-insert/update data into DART.

My original intention was to have this migration replace the existing MLWH one and do both MLWH and DART updates on failure, but I think we'd be best keeping the separate MLWH and DART migrations. What do you think? This will impact on the SQL query with which you update MLWH

Reply:
I still think it's a good idea for this to replace the MLWH migration but let's do that once we're happy with how it performs with a live DART DB?